templates/tfengine/components/cicd/main.tf

```diff
@@ -134,6 +134,22 @@ resource "google_project_iam_member" "cloudbuild_builds_editors" {
     google_project_service.services,
   ]
 }
+
+# IAM permission to allow approvers to trigger builds.
```

Comment:
IAM permission to allow approvers to impersonate the Cloud Build user-specified Service Account.

Reply:
Updated.